stories/4-components/modals.stories.js

```diff
@@ -70,5 +87,13 @@ export const opened = () => (
 )
 
 opened.story = {
-  parameters: { layout: { height: '80vh' } },
+  parameters: { 
+    layout: { height: '80vh' },
+    info: {
+      text: `
+        ### Opened
+        Opned modal is a container to display content in an overlay window.
```

Comment:
@singhsrm `Opened` is misspelled.

Reply:
@singhsrm `Opened` is still misspelled. Look on line 95. You have `Opned`. Should be `Opened`